templates/admin/reporting/nav-filters.php

```diff
@@ -76,29 +76,28 @@
 
 			<li class="llms-nav-item llms-analytics-form">
 
-				<label><?php _e( 'Courses', 'lifterlms' ); ?></label>
-
-				<select class="llms-select2-post" data-placeholder="<?php _e( 'Filter by Course(s)', 'lifterlms' ); ?>" data-post-type="course" id="llms-course-ids-filter" name="course_ids[]" multiple="multiple">
+				<label><?php esc_html_e( 'Courses', 'lifterlms' ); ?></label>
+				<select data-post-statuses="<?php echo esc_attr( implode( ',', array_keys( get_post_statuses() ) ) ); ?>" class="llms-select2-post" data-placeholder="<?php esc_html_e( 'Filter by Course(s)', 'lifterlms' ); ?>" data-post-type="course" id="llms-course-ids-filter" name="course_ids[]" multiple="multiple">
 					<?php foreach ( $current_courses as $course_id ) : ?>
-						<option value="<?php echo $course_id; ?>" selected><?php echo get_the_title( $course_id ); ?> <?php printf( __( '(ID# %d)', 'lifterlms' ), $course_id ); ?></option>
+						<option value="<?php echo esc_attr( $course_id ); ?>" selected><?php echo get_the_title( $course_id ); ?> <?php printf( __( '(ID# %d)', 'lifterlms' ), $course_id ); ?></option>
```

Comment:
``` printf( __( '(ID# %d)', 'lifterlms' ), $course_id );```
that translation needs to be escaped, also can you add `Translators` suggestions? E.g.
https://github.com/gocodebox/lifterlms/blob/94903b54585ebd572e533d25698fd9b1aa2cb347/templates/loop/enroll-date.php#L23